.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+text.md
```

Comment:
- [ ] based on the requirement highlighted below you are supposed to ignore the file `test.md` not `text.md` kindly correct this typographical error :+1: 
- A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.